endpoints/information/src/main.cpp

```diff
@@ -34,18 +34,11 @@ namespace irods::http::handler
 			res.set(field_type::content_type, "application/json");
 			res.keep_alive(_req.keep_alive());
 
-#ifdef IRODS_ENABLE_GENQUERY2
-#  define GENQUERY2_ENABLED true
-#else
-// NOLINTNEXTLINE(cppcoreguidelines-macro-usage)
-#  define GENQUERY2_ENABLED false
-#endif // IRODS_ENABLE_GENQUERY2
-
 			// clang-format off
 			res.body() = json{
 				{"api_version", irods::http::version::api_version},
 				{"build", irods::http::version::sha},
-				{"genquery2_enabled", GENQUERY2_ENABLED},
+				{"genquery2_enabled", true},
```

Comment:
Should we deprecate this field?

Reply:
Whether we should deprecate or remove it isn't clear to me yet. I'm trying to think though whether there's value in keeping it.

Are users relying on it? What if GenQuery2 contains a nasty bug that forces us to disable it in the HTTP API? How do we communicate that? Should it be communicated through that property or just documentation?

The fact we haven't reached 1.0 status already provides signal that things can change a lot. Does that mean we can skip deprecating things? I'm pretty sure the interface has changed without notice in the past.

Reply:
After some discussion, we're going to remove that property.